src/index.ts

```diff
@@ -144,6 +150,17 @@ class Client {
             metric += '|@' + sampling;
         }
 
+        if (this.tags) {
+            if (Array.isArray(this.tags)) {
+                metric += `|#${this.tags.join(',')}`;
+            } else {
+                const tags = Object.keys(this.tags)
+                    .map((tag) => `${tag}:${this.tags[tag]}`)
```

Comment:
The DataDog documentation indicates that you can mix simple tags with tag/value pairs. While I don't have a personal need for this feature, it could be beneficial for other users. We can easily support this functionality by adding a simple check to see if the value is empty and then rendering only the tag. WDYT?

Reply:
Good catch; I've updated this to support simple tags

Reply:
LGTM thank you!